template/menu.mustache

```diff
@@ -61,10 +61,10 @@
 				{{/twitch}}
 			{{/user}}
 			{{#cookies.darkmode}}
-				<li><a href='?{{{forms.get.page}}}={{{forms.pages.main}}}&{{{forms.get.dark_mode}}}=0'>Enable light mode</a></li>
+				<!-- <li><a href='?{{{forms.get.page}}}={{{forms.pages.main}}}&{{{forms.get.dark_mode}}}=0'>Enable light mode</a></li> -->
 			{{/cookies.darkmode}}
 			{{^cookies.darkmode}}
-				<li><a href='?{{{forms.get.page}}}={{{forms.pages.main}}}&{{{forms.get.dark_mode}}}=1'>Enable dark mode</a></li>
+				<!-- <li><a href='?{{{forms.get.page}}}={{{forms.pages.main}}}&{{{forms.get.dark_mode}}}=1'>Enable dark mode</a></li> -->
 			{{/cookies.darkmode}}
```

Comment:
these should be outright deleted - git handles a memory of old code, so if we ever need it again it's always there, but 99.9% of the time we don't so makes sense to delete it by default

Reply:
This doesn't need to be done as part of this PR but can be fixed as part of a general cleanup when the UX Design 2 milestone comes to a close. But as there's a conflict with this PR it may as well be fixed now.